system-tests/src/main/java/io/apicurio/registry/systemtests/client/ApicurioRegistryApiClient.java

```diff
@@ -48,6 +49,7 @@ public boolean isServiceAvailable() {
 
         // Get request builder
         HttpRequest.Builder requestBuilder = HttpClientUtils.newBuilder()
+                .version(HttpClient.Version.HTTP_1_1)
```

Comment:
I set `HttpClient.Version.HTTP_1_1` in `HttpClientUtils.newBuilder()`, see https://github.com/obabec/apicurio-registry/blob/testsuite-upstream/system-tests/src/main/java/io/apicurio/registry/systemtests/framework/HttpClientUtils.java#L29.